service/src/main/java/uk/nhs/adaptors/gp2gp/sds/builder/SdsRequestBuilder.java

```diff
@@ -37,28 +37,30 @@
         "urn:nhs:names:services:gpconnect:documents:fhir:rest:read:binary-1";
 
     private static final String API_KEY_HEADER = "apikey";
+    private static final String X_CORRELATION_ID_HEADER = "X-Correlation-Id";
```

Comment:
This class is being removed in this PR: https://github.com/nhsconnect/integration-adaptor-gp2gp/pull/129. I think the changes should be only done in a consumer.